sbin/create_course.sh

```diff
@@ -35,10 +35,7 @@ echo $DATABASE_COURSE_USER
 ########################################################################################################################
 ########################################################################################################################
 
-CONN_STRING="-h ${DATABASE_HOST} -U ${DATABASE_USER}"
-if [ -d "${DATABASE_HOST}" ]; then
-    CONN_EXTRA="${CONN_STRING} -p ${DATABASE_PORT}"
-fi
+CONN_STRING="-h ${DATABASE_HOST} -U ${DATABASE_USER} -p ${DATABASE_PORT}"
 
```

Comment:
Does this still work fine if we use a unix socket or regular network connection? As well for the setup_sample_courses script too.

Reply:
Yes — the default `host` parameter for `psql` is `/var/run/postgresql`, and it also supports URL.
For `setup_sample_courses.py`, the [scheme](https://www.postgresql.org/docs/current/libpq-connect.html#id-1.7.3.8.3.6) `user:pass@host:port/dbname` needs the socket `host` to be percent-encoded.  So `/dbname?parameters=` seems to be more flexible.